content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,634 @@
+appear in the list by default. If the :guilabel:`To Approve` filter is removed, then all statuses
+appear. :guilabel:`Approved` requests are highlighted in green, :guilabel:`To Submit` (drafts)
+requests are highlighted in blue, and the :guilabel:`Refused` requests are highlighted in gray.
+
+To approve a time off request, click :guilabel:`👍 Approve` at the end of the line, to refuse a
+request, click :guilabel:`X Refuse`.
+
+.. image:: time_off/time-off-requests.png
+   :align: center
+   :alt: Time off requests with the filter, groupings, and status sections highlighted.
+
+If more details are needed, click anywhere on the time off request line (except for :guilabel:`👍
+Approve` and :guilabel:`X Refuse`) to load the time off request form. Depending on the rights of the
+user, changes can be made. To modify the request, click the :guilabel:`Edit` button, make any
+desired changes, then click :guilabel:`Save`. It is also possible to approve or refuse the request
+from this form. Click the :guilabel:`Approve` button to approve, or the :guilabel:`Refuse` button to
+refuse the request.
+
+My time off
+===========
+
+The :guilabel:`My Time Off` section of the *Time Off* application contains the time off dashboard,
+as well as the user's time off requests and allocations.
+
+.. _time_off/dashboard:
+
+Dashboard
+---------
+
+All users have access to the time off dashboard, which is the default view in the *Time Off*
+application. The dashboard can also be accessed at any point in the application by navigating to
+:menuselection:`Time Off application --> My Time Off --> Dashboard`.
+
+The current year is displayed, and the current day is highlighted in red. To change the view, click
+on the desired button at the top. Options are :guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`,
+or :guilabel:`Year` (the default). To change the presented dates, click the left and right arrows on
+either side of the :guilabel:`Today` button. The calendar view adjusts in increments of the
+presented view. For example, if :guilabel:`Week` is selected, the arrows adjust the view by one
+week. To change the view at any point to a view that includes the current date, click the
+:guilabel:`Today` button.
+
+Above the calendar view is a summary of the users time off balances. Every time off type that has
+been allocated appears in its own summary box. Each summary lists the type of time off, the
+corresponding icon, the current available balance (in hours or days), and an expiration date (if
+applicable). To view more details on the specific time off, click the :guilabel:`? (question mark)`
+after the line `Hours (or Days) Available`. A pop up appears listing the total time allocated, the
+approved time off, the planned time off, and the total ending balance with those numbers factored
+in. The time off balance labeled as :guilabel:`Left` is what is displayed next to the icon in the
+summary. Click the :guilabel:`? (question mark)` again to close the pop-up.
```

Comment:
I'm actually not seeing any such question mark...is this an aspect that requires Developer Mode, perhaps? Either way, I think it would be best to double-check if it's around in all 15.0 dbs, because the runbot I'm using currently does offer this functionality

Reply:
You are correct- it only appears in the 15.2 docs. I'll remove this!